codice/ddf
# DDF-4226 Add ability to identify country features while parsing geo.json

#### What does this PR do?
Adds customized ability to determine if an incoming geo.json feature contains the geometry of a country by parsing additional property field.

#### Who is reviewing it? 
@bdeining @djblue @gjvera @rymach 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@djblue 
@millerw8

#### How should this be tested?
- Full build
- Ingest sample geo.json data: execute command `gazetteer:update "$ddf.data/countries.geo.json"` to ingest the default countries.geo.json file.
- Perform a basic search in Intrigue, and search for `Somewhere Specific`
- For the keyword, search for one of the countries provided in the geo.json file
- Verify that the country is outlined in intrigue, and that the coordinates are correct in the coordinate field of the search
- Ingest GeoNames data and verify no regressions

#### Any background context you want to provide?

#### What are the relevant tickets?
[DDF-4226](https://codice.atlassian.net/browse/DDF-4226)

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
